src/d3/hapticplot.d3.spec.ts

```diff
@@ -53,3 +87,44 @@ function getPosition(element: HTMLElement, shape: string): Array<{x: number, y:
   }
   return positionArray;
 }
+
+// returns array of each generated objects color
+function getColor(element: HTMLElement, shape: string): Array<string>{
+  const childrenArray = element.querySelectorAll(shape);
```

Comment:
as SVGElement[] would prevent the casts to `any` later on.  In general, avoid `any` at all costs (you can use `unknown` where it makes sense, or cast it to what you need)

Reply:
Or do what Natalie suggested, that's even better

Reply:
done!